elk/metrics/eval.py

```diff
@@ -23,6 +23,8 @@ class EvalResult:
     roc_auc: RocAucResult
     """Area under the ROC curve. For multi-class classification, each class is treated
     as a one-vs-rest binary classification problem."""
+    cal_thresh: float | None
```

Comment:
Any reason why we are adding cal_trhesh to the eval results :green_circle: ?

Reply:
i just thought it would be interesting to see 🤷‍♂️ and seemed easy